### PR TITLE
fix(table): fix invalid table component index

### PR DESCRIPTION
**Pull request by yousifalraheem**
### This PR addresses the following:
- Fix invalid **Table** component index
- Remove invalid `async` from `describe` statement in **Table** component test
- Remove unnecessary console logs in **Table** component test
- Fix sub-dependency vulnerabilities

### CODE REVIEW

File: src/Table/Table.test.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ describe("Component: Table", () => {
     });
 
     it("Should render and be able to sort rows ", async () => {
-        const event: jest.Mock = jest.fn((rows: Array<TableRow>, sortByColumn: TableHeader) => console.log("onAfterSorting called"));
+        const event: jest.Mock = jest.fn((rows: Array<TableRow>, sortByColumn: TableHeader) => true);
```
Comment (kherP) [inline src/Table/Table.test.tsx:85]: i think this can be simplified with `jest.fn()`

File: src/Table/Table.test.tsx [hunk 2]
```diff
@@ -144,7 +144,7 @@ describe("Component: Table", () => {
     it("Should render and do pagination where necessary ", async () => {
         const pageSize: number = 30;
         const paginationValue: number = 1;
-        const setPage: jest.Mock = jest.fn((n: number) => console.log("setPage called"));
+        const setPage: jest.Mock = jest.fn((n: number) => true);
```
Comment (kherP) [inline src/Table/Table.test.tsx:147]: i think this can be simplified with `jest.fn()`

File: develop/components/pages/TablePage.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import { Table, Column, TableRow, PrimaryActionButton, ActionLinkItem, TableHeader, DataItem, FilterItem, FilterProps, EditProps, EditMode } from "../../../src/Table";
+import { Table, Column, TableRow, PrimaryActionButton, ActionLinkItem, TableHeader, DataItem, FilterItem, FilterProps, EditProps, EditMode } from "../../../src/Table/Table";
```
Comment (mario-subo) [inline develop/components/pages/TablePage.tsx:2]: the Table folder has an index.ts with `export * from "./Table";` so I'm not sure why you need this change. Does it not work? If not you should remove the index.ts file or fix it there.

File: src/Table/Table.test.tsx [hunk 4]
```diff
@@ -168,7 +168,7 @@ describe("Component: Table", () => {
     });
 
     it("Should render and be able to expand the subRows and row details ", async () => {
-        const onRowExpanded: jest.Mock = jest.fn((rows: Array<TableRow>) => console.log("onRowExpanded called"));
+        const onRowExpanded: jest.Mock = jest.fn((rows: Array<TableRow>) => true);
```
Comment (kherP) [inline src/Table/Table.test.tsx:171]: so as this

File: src/Table/Table.test.tsx [hunk 5]
```diff
@@ -499,9 +499,7 @@ describe("Component: Table", () => {
             results = rows;
         });
 
-        const onRemoveFilter: jest.Mock = jest.fn((filterItem: FilterItem) => {
-            console.log(filterItem);
-        });
+        const onRemoveFilter: jest.Mock = jest.fn((filterItem: FilterItem) => true);
```
Comment (kherP) [inline src/Table/Table.test.tsx:502]: and this

File: src/Table/Table.test.tsx [hunk 6]
```diff
@@ -262,7 +262,7 @@ describe("Component: Table", () => {
         expect(onRowSelected).toHaveBeenCalledTimes(4);
     });
 
-    describe("should handle inline textbox edit ", async () => {
+    describe("should handle inline textbox edit ", () => {
```
Comment (mario-subo) [inline src/Table/Table.test.tsx:265]: 😏

Comment (yousifalraheem) [reply src/Table/Table.test.tsx:265]: `async` is not supported for `describe` statements

File: develop/components/pages/TablePage.tsx (hunk 3 above)
Comment (yousifalraheem) [reply develop/components/pages/TablePage.tsx:2]: I replaced the `export * from "./Table";`  with `export { Table } from "./Table";` to make it consistent with the rest of the components

File: src/Table/Table.test.tsx (hunk 2 above)
Comment (yousifalraheem) [reply src/Table/Table.test.tsx]: Done

File: src/Table/Table.test.tsx (hunk 4 above)
Comment (yousifalraheem) [reply src/Table/Table.test.tsx]: Done

File: src/Table/Table.test.tsx (hunk 5 above)
Comment (yousifalraheem) [reply src/Table/Table.test.tsx]: Done

Comment (mario-subo) [reply src/Table/Table.test.tsx]: you should resolve the conversation if it's done. KR or Yousif

File: develop/components/pages/TablePage.tsx (hunk 3 above)
Comment (mario-subo) [reply develop/components/pages/TablePage.tsx:2]: Oh ok it's like that in all the components I see. Maybe something we can refactor later.

File: src/Table/Table.test.tsx (hunk 6 above)
Comment (mario-subo) [reply src/Table/Table.test.tsx:265]: 😂

File: src/Table/Table.test.tsx (hunk 1 above)
Comment (yousifalraheem) [reply src/Table/Table.test.tsx]: Done

**Pull request merged**